packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -172,4 +129,11 @@ class Product extends Component {
     }
 }
 
-export default classify(defaultClasses)(Product);
+export default connect(
+    ({ cart: { guestCartId } = {} }) => ({ guestCartId }),
+    {
+        addItemToCart,
+        getCartDetails,
+        toggleCart
+    }
+)(Product);
```

Comment:
Please consider defining `mapStateToProps` and `mapDispatchToProps` just before this export, as we've done in the other components. It's a bit less terse but more clear.